frontend/package.json

```diff
@@ -9,8 +9,8 @@
   ],
   "scripts": {
     "clean": "rm -rf ./public/dist",
-    "dev": "yarn clean && ts-node -O '{\"module\":\"commonjs\"}' ./node_modules/.bin/webpack --mode development --watch --progress",
-    "build": "yarn clean && NODE_ENV=production ts-node -O '{\"module\":\"commonjs\"}' ./node_modules/.bin/webpack --mode=production",
+    "dev": "yarn clean && NODE_OPTIONS=--max-old-space-size=4096 ts-node -O '{\"module\":\"commonjs\"}' ./node_modules/.bin/webpack --mode development --watch --progress",
```

Comment:
What's up with these warnings when running `yarn run dev`?
```
WARNING in ./node_modules/monaco-editor-core/esm/vs/editor/common/services/editorSimpleWorker.js 374:12-383:17
Critical dependency: the request of a dependency is an expression
```

Reply:
> What's up with these warnings when running `yarn run dev`?
> 
> ```
> WARNING in ./node_modules/monaco-editor-core/esm/vs/editor/common/services/editorSimpleWorker.js 374:12-383:17
> Critical dependency: the request of a dependency is an expression
> ```

@JPinkney ?

Reply:
This looks relevant: https://github.com/Microsoft/monaco-editor-webpack-plugin/issues/13

Reply:
That was the only solution I could find as well. I'll try it out and report back